src/frontend/src/apis/transformers/change-requests.transformers.ts

```diff
@@ -12,15 +12,18 @@ import { ChangeRequest, ChangeRequestType, ImplementedChange } from 'shared';
  * @returns Properly transformed change request object.
  */
 export const changeRequestTransformer = (changeRequest: ChangeRequest) => {
-  const data: any = {
+  interface MaybeActiveChangeRequest extends ChangeRequest {
```

Comment:
I forgot this, but can we define this interface before changeRequestTransformer definition? (Before line 14)